### PR TITLE
Feature/code cleanup and folder restructuring

### PR DESCRIPTION
**Pull request by saschadoemer**

### CODE REVIEW

File: agrirouter-api-dotnet-standard-test/Service/Onboard/OnboardServiceTest.cs [hunk 1]
```diff
@@ -11,14 +11,34 @@
...
+    ///     Functional tests.
     /// </summary>
     [Collection("Integrationtest")]
     public class OnboardServiceTest : AbstractIntegrationTest
     {
         private static readonly UtcDataService UtcDataService = new UtcDataService();
         private static readonly HttpClient HttpClient = HttpClientFactory.HttpClient();
 
+        [Fact]
+        public void GivenInvalidRequestTokenWhenOnboardingThenThereShouldBeAValidResponse()
```
Comment (Frank-Wiebeler) [inline agrirouter-api-dotnet-standard-test/Service/Onboard/OnboardServiceTest.cs:23]: Wouldn't GivenInvalidRequestTokenWhenOnboardingThenThereShouldBeAnException() be more accurate?

File: agrirouter-api-dotnet-standard-test/Service/Onboard/SecuredOnboardingServiceTestForFarmingSoftware.cs [hunk 2]
```diff
@@ -10,13 +10,34 @@
...
-    /// Functional tests.
+    ///     Functional tests.
     /// </summary>
     [Collection("Integrationtest")]
     public class SecuredOnboardingServiceTestForFarmingSoftware : AbstractSecuredIntegrationTestForFarmingSoftware
     {
         private static readonly HttpClient HttpClient = HttpClientFactory.HttpClient();
 
+        [Fact]
+        public void GivenInvalidRequestTokenWhenOnboardingThenThereShouldBeAValidResponse()
```
Comment (Frank-Wiebeler) [inline agrirouter-api-dotnet-standard-test/Service/Onboard/SecuredOnboardingServiceTestForFarmingSoftware.cs:21]: Wouldn't GivenInvalidRequestTokenWhenOnboardingThenThereShouldBeAnException() be more accurate?

File: agrirouter-api-dotnet-standard-test/Service/Onboard/SecuredOnboardingServiceTestForTelemtryPlatform.cs [hunk 3]
```diff
@@ -10,63 +10,63 @@
...
+    ///     Functional tests.
     /// </summary>
     public class SecuredOnboardingServiceTestForTelemtryPlatform : AbstractSecuredIntegrationTestForTelemetryPlatform
     {
         private static readonly HttpClient HttpClient = HttpClientFactory.HttpClient();
 
-        [Fact(Skip = "Will not run successfully without changing the registration code.")]
-        public void GivenValidRequestTokenWhenOnboardingThenThereShouldBeAValidResponse()
+        [Fact]
+        public void GivenInvalidRequestTokenWhenOnboardingThenThereShouldBeAValidResponse()
```
Comment (Frank-Wiebeler) [inline agrirouter-api-dotnet-standard-test/Service/Onboard/SecuredOnboardingServiceTestForTelemtryPlatform.cs:20]: Wouldn't GivenInvalidRequestTokenWhenOnboardingThenThereShouldBeAnException() be more accurate?

File: agrirouter-api-dotnet-standard-test/Service/Onboard/SecuredOnboardingServiceTestForTelemtryPlatform.cs [hunk 4]
```diff
@@ -10,63 +10,63 @@
...
         {
             var onboardingService =
                 new SecuredOnboardingService(Environment, HttpClient);
 
             var parameters = new OnboardParameters
             {
-                Uuid = Guid.NewGuid().ToString(),
+                Uuid = GetType().FullName,
```
Comment (Frank-Wiebeler) [inline agrirouter-api-dotnet-standard-test/Service/Onboard/SecuredOnboardingServiceTestForTelemtryPlatform.cs:27]: Why aren't you using Guid.NewGuid().ToString() here? The test is about an invalid Registration code but you add an additional potential issue?

File: agrirouter-api-dotnet-standard-api/Logging/LoggingHandler.cs [hunk 5]
```diff
@@ -30,19 +30,13 @@ public LoggingHandler(HttpMessageHandler innerHandler)
         {
             Log.Debug("Request:");
             Log.Debug(request.ToString());
-            if (request.Content != null)
-            {
-                Log.Debug(await request.Content.ReadAsStringAsync());
-            }
+            if (request.Content != null) Log.Debug(await request.Content.ReadAsStringAsync());
```
[the remark this answers is not in the document]
Comment (saschadoemer) [reply agrirouter-api-dotnet-standard-api/Logging/LoggingHandler.cs:33]: Common .NET coding style, default setting for code optimization in Rider.

File: agrirouter-api-dotnet-standard-impl/Service/Messaging/Abstraction/SendMessageBaseService.cs [hunk 6]
```diff
@@ -103,18 +134,15 @@ public bool MessageHasToBeChunked(string base64MessageContent)
...
+        /// <param name="chunkSize">Size of the chunks, <seealso cref="ChunkSizeDefinition" /></param>
         /// <returns>-</returns>
         /// <exception cref="ChunkSizeNotSupportedException">-</exception>
         public IEnumerable<string> ChunkMessageContent(string base64MessageContent, int chunkSize)
         {
-            if (chunkSize > ChunkSizeDefinition.MaximumSupported)
-            {
-                throw new ChunkSizeNotSupportedException();
-            }
+            if (chunkSize > ChunkSizeDefinition.MaximumSupported) throw new ChunkSizeNotSupportedException();
```
[the remark this answers is not in the document]
Comment (saschadoemer) [reply agrirouter-api-dotnet-standard-impl/Service/Messaging/Abstraction/SendMessageBaseService.cs:145]: Common .NET coding style, default setting for code optimization in Rider.

File: agrirouter-api-dotnet-standard-api/Logging/LoggingHandler.cs [hunk 7]
```diff
@@ -30,19 +30,13 @@ public LoggingHandler(HttpMessageHandler innerHandler)
...
 
             var response = await base.SendAsync(request, cancellationToken);
 
             Log.Debug("Response:");
             Log.Debug(response.ToString());
-            if (response.Content != null)
-            {
-                Log.Debug(await response.Content.ReadAsStringAsync());
-            }
+            if (response.Content != null) Log.Debug(await response.Content.ReadAsStringAsync());
```
[the remark this answers is not in the document]
Comment (saschadoemer) [reply agrirouter-api-dotnet-standard-api/Logging/LoggingHandler.cs:39]: Common .NET coding style, default setting for code optimization in Rider.

File: agrirouter-api-dotnet-standard-impl/Service/Messaging/Abstraction/SendMultipleMessagesBaseService.cs [hunk 8]
```diff
@@ -67,20 +69,15 @@ public EncodedMessage Encode(MultipleMessageEntry multipleMessageEntry)
...
         /// <returns></returns>
         public void CheckWetherMessageShouldHaveBeenChunked(SendMessageParameters sendMessageParameters)
         {
             var base64MessageContent = sendMessageParameters.Base64MessageContent;
             var byteCount = Encoding.Unicode.GetByteCount(base64MessageContent);
-            if (byteCount / ChunkSizeDefinition.MaximumSupported > 1)
-            {
-                throw new MessageShouldHaveBeenChunkedException();
-            }
+            if (byteCount / ChunkSizeDefinition.MaximumSupported > 1) throw new MessageShouldHaveBeenChunkedException();
```
[the remark this answers is not in the document]
Comment (saschadoemer) [reply agrirouter-api-dotnet-standard-impl/Service/Messaging/Abstraction/SendMultipleMessagesBaseService.cs:80]: Common .NET coding style, default setting for code optimization in Rider.

File: agrirouter-api-dotnet-standard-impl/Service/Messaging/Vcu/OffboardVcuService.cs [hunk 9]
```diff
@@ -61,10 +61,7 @@ public EncodedMessage Encode(OffboardVcuParameters offboardVcuParameters)
             };
 
             var offboardingRequest = new OffboardingRequest();
-            foreach (var endpoint in offboardVcuParameters.Endpoints)
-            {
-                offboardingRequest.Endpoints.Add(endpoint);
-            }
+            foreach (var endpoint in offboardVcuParameters.Endpoints) offboardingRequest.Endpoints.Add(endpoint);
```
[the remark this answers is not in the document]
Comment (saschadoemer) [reply agrirouter-api-dotnet-standard-impl/Service/Messaging/Vcu/OffboardVcuService.cs:64]: Common .NET coding style, default setting for code optimization in Rider.

File: agrirouter-api-dotnet-standard-impl/Service/Messaging/Vcu/OnboardVcuService.cs [hunk 10]
```diff
@@ -62,9 +62,7 @@ public EncodedMessage Encode(OnboardVcuParameters onboardVcuParameters)
 
             var onboardingRequest = new OnboardingRequest();
             foreach (var onboardingRequestEntry in onboardVcuParameters.OnboardingRequests)
```
[the remark this answers is not in the document]
Comment (saschadoemer) [reply agrirouter-api-dotnet-standard-impl/Service/Messaging/Vcu/OnboardVcuService.cs:64]: Common .NET coding style, default setting for code optimization in Rider.

**Pull request merged**